src/registrar.js

```diff
@@ -198,13 +200,20 @@ const getEntry = async name => {
       // Owned
       ret.state = 2
     }
-    if (permEntry.ownerOf) {
-      ret.isNewRegistrar = true
-      ret.registrant = permEntry.ownerOf
-    }
     if (permEntry.nameExpires) {
       ret.expiryTime = permEntry.nameExpires
     }
+    if (permEntry.ownerOf) {
+      ret.registrant = permEntry.ownerOf
+      ret.isNewRegistrar = true
+    }else if (permEntry.nameExpires && permEntry.gracePeriod){
```

Comment:
What's the point of the check for `gracePeriod` here? Won't it always be true, since it's a constant?